tools/android/packaging/xbmc/src/XBMCJsonRPC.java.in

```diff
@@ -584,9 +584,9 @@ public class XBMCJsonRPC
             if (movie.has("fanart") && !movie.get("fanart").getAsString().isEmpty())
               notificationBuilder.setBackground(XBMCFileContentProvider.buildUri(
                       getDownloadUrl(movie.get("fanart").getAsString())).toString());
-            if (movie.has("thumbnail") && !movie.get("thumbnail").getAsString().isEmpty())
+            if (movie.has("poster") && !movie.get("poster").getAsString().isEmpty())
```

Comment:
I doubt this exists in the movie object. You might need to check "art" and then "poster" as a member of "art"